src/coreclr/src/vm/diagnosticsprotocol.h

```diff
@@ -70,6 +70,7 @@ namespace DiagnosticsIpc
         Dump          = 0x01,
         EventPipe     = 0x02,
         Profiler      = 0x03,
+        Process       = 0x04,
```

Comment:
I wanted to keep the Command Set generic in case we decide to add other commands.  For instance, I was talking with Noah, and he posited that ResumeRuntime should be under something besides the Server command set.  I was considering putting the ResumeRuntime command under this Process command set.